solr/server/resources/log4j2.xml

```diff
@@ -67,6 +67,10 @@
     <Logger name="org.apache.solr.core.SolrCore.SlowRequest" level="info" additivity="false">
       <AppenderRef ref="SlowFile"/>
     </Logger>
+    <Logger name="org.apache.solr.update.processor.LogDeleteQueryProcessorFactory" level="warn"/>
+    <Logger name="org.apache.solr.update.processor.LogDeleteQueryProcessorFactory.delete" level="info" additivity="false">
```

Comment:
Is LogDeleteQueryProcessorFactory defined somewhere? Or should the class name be `LogUpdateProcessorFactory` ?

Reply:
Yup, it should have been LogUpdateProcessorFactory itself, since the changes are planned in LogUpdateProcessorFactory class itself. I have made the changes accordingly.